simbaa1/Politico
# #163712516 Set up Get Office by its id

*What does this PR do?*  
Add api endpoint for getting a party by its id

*Description of Task to be completed?*

- Getting a single party by sending a GET request through /api/v1/offices/id
- Response payload should be, if successful:
`{
“status” : Integer ,
“data” : [ {
“id” : Integer ,
“name” : String ,
"type": String
} ]
}`

*How should this be manually tested?*  
-Clone the repo
-install an set up virtual environment.
-Install requirements through pip install -r requirements.txt 
-On terminal fire up the server through python run.py
- Test it on API Client like POSTMAN

*What are the relevant pivotal tracker stories?*  
https://www.pivotaltracker.com/story/show/*What does this PR do?*  
Add api endpoint for creating parties

*Description of Task to be completed?*

- Creation a new party by sending a POST request through /api/v1/parties
- Response payload should be, if successful:
`{
“status” : Integer ,
“data” : [ {
“id” : Integer ,
“name” : String ,
"chairperson": "chairperson"
} ]
}`

*How should this be manually tested?*  
-Clone the repo
-install an set up virtual environment.
-Install requirements through pip install -r requirements.txt 
-On terminal fire up the server through python run.py
- Test it on API Client like POSTMAN

*What are the relevant pivotal tracker stories?*  
https://www.pivotaltracker.com/story/show/163711932


Screenshot
![image](https://user-images.githubusercontent.com/26428556/52488097-2fffdc80-2bd0-11e9-9a33-c3559c34a019.png)
